table/tables/index.go

```diff
@@ -233,8 +233,8 @@ func (c *index) Create(sctx sessionctx.Context, rm kv.RetrieverMutator, indexedV
 		// If the index kv was untouched(unchanged), and the key/value already exists in mem-buffer,
 		// should not overwrite the key with un-commit flag.
 		// So if the key exists, just do nothing and return.
-		_, err = txn.GetMemBuffer().Get(ctx, key)
-		if err == nil {
+		v, err := txn.GetMemBuffer().Get(ctx, key)
+		if err == nil && len(v) != 0 {
```

Comment:
a bug fix?

Reply:
The old `txn.GetMemBuffer()` returns a `BufferStore` which already filtered out the tombstones.